fix: set QR code background to always be white

Robot-Inventor opened the pull request:
Keeping the QR code background white in dark mode is preferable for several reasons:

- Some QR code scanners may fail to read inverted (black background) QR codes
- On glossy LCDs, reflections on dark areas of the screen can cause QR code scanning to fail

GeorgeHulpoi commented:
@Robot-Inventor
- You can create a field called `forceWhiteBackgroundOnQrCode` and include it in the documentation. 
- Add `border-radius: 6px` to the image.
- If you add `padding`, you should also increase the `width` and `height`.
- You have to add an end-to-end test. This should be simple: you just need to check whether setting the configuration field is reflected in the DOM structure.

Robot-Inventor commented:
@GeorgeHulpoi 
I added the `forceWhiteBackgroundOnQrCode` option and fixed the issues you pointed out.

I also added tests. Since I couldn’t run the existing tests locally, I added `type` to some imports.

Robot-Inventor commented:
@GeorgeHulpoi It looks like you may have already merged the main branch to this pr on your side.
